drivers/hmis/app/models/hmis/ce/referral_message_handler.rb

```diff
@@ -114,6 +126,14 @@ def set_custom_referral_status(status_key:)
       referral.update!(custom_status: status)
     end
 
+    def disable_step(node_id:)
```

Comment:
This is a new `disable_step` side effect. This lets one task specify that when this task has an event, some _other_ task should change status. The trigger config for this looks like:
```
trigger_config: [
  {
    event: 'complete_step',
    message: 'disable_step',
    params: { 'node_id': other_task.id },
  },
],
```

To discuss: I'm not totally sold on doing this by `node_id`, I guess because it makes the workflow definition more opaque, and maybe these `node_id`s are more changeable? Another possibility is to add a new `key` column which could be a human readable string, sort of equivalent to form IDs vs. `definition_identifier`s.